core/src/main/scala/kafka/api/TopicMetadata.scala

```diff
@@ -61,25 +62,25 @@ case class TopicMetadata(topic: String, partitionsMetadata: Seq[PartitionMetadat
     val topicMetadataInfo = new StringBuilder
     topicMetadataInfo.append("{TopicMetadata for topic %s -> ".format(topic))
     errorCode match {
-      case ErrorMapping.NoError =>
+      case e if e == Errors.NONE.code =>
```

Comment:
Re; my previous comment, this match would be more natural if we convert the error code to an Errors instance.